### ISSUE TITLE
Aqueduct document fails with missing getter "summary"

### ISSUE THREAD

**Issue by gokr** [stablekernel/aqueduct#550]
Below is a partial trace, not sure what can be wrong?

```
gokr@maz:~/bio/cloud/transformer$ aqueduct document
-- Aqueduct CLI Version: 3.0.2
-- Aqueduct project version: 3.0.1
*** Uncaught error
    Bad state: NoSuchMethodError: The getter 'summary' was called on null.
Receiver: null
Tried calling: summary
  **** Stacktrace
  * #0      Object.noSuchMethod (dart:core/runtime/libobject_patch.dart:46:5)
  * #1      ResourceController.documentOperations.<anonymous closure>.<anonymous closure> (package:aqueduct/src/http/resource_controller.dart:281:50)
  * <asynchronous suspension>
  * #2      APIDocumentContext.finalize.<anonymous closure> (package:aqueduct/src/openapi/documentable.dart:227:41)
  * #3      Future.forEach.<anonymous closure> (dart:async/future.dart:486:26)
  * #4      Future.doWhile.<anonymous closure> (dart:async/future.dart:528:26)
  * #5      _RootZone.runUnaryGuarded (dart:async/zone.dart:1314:10)
  * #6      _RootZone.bindUnaryCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1353:26)
  * #7      _RootZone.runUnary (dart:async/zone.dart:1379:54)
  * #8      _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
  * #9      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
  * #10     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
  * #11     Future._complete (dart:async/future_impl.dart:476:7)
  * #12     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
  * #13     _AsyncAwaitCompleter.complete (dart:async/runtime/libasync_patch.dart:28:18)
  * #14     _completeOnAsyncReturn (dart:async/runtime/libasync_patch.dart:292:13)
  * #15     ManagedEntity.documentComponents.<anonymous closure>.<anonymous closure> (package:aqueduct/src/db/managed/entity.dart)
  * <asynchronous suspension>
```

**Comment by joeconwaystk** [comment]
Per your other info, this looks like a bug with ManagedObjectControllers are auto-documenting. More generally, a bug with any subclass of a subclass of ResourceController. This will get fixed in 3.0.2.

**Comment by joeconwaystk** [comment]
@gokr Are you able to verify this is fixed in 3.0.2?
